### DISCUSSION TITLE
Hi, how can solve abc.com/products/ not found

### DISCUSSION THREAD

**Discussion by IRediTOTO** [vercel/next.js#12980] [Help]
# Bug report

## Describe the bug

Hi, how can solve link with / ?
Example:
I goto http://localhost:3000/products => It will work
But with http://localhost:3000/products/ => not found :s
after products/[item-id-can-be-here]
Is there any idea to solve all link for this? 
## Expected behavior

goto url abc.com/products**/** and we will get abc.com/products without error 404

**Comment by RomanKiyashev** [comment]
@warcraft14115 https://github.com/zeit/next.js/issues/8913

**Comment by IRediTOTO** [comment]
> @warcraft14115 #8913
@RomanKiyashev1 
It will work , but still bug I think. Title of page still 404
![image](https://user-images.githubusercontent.com/15695252/82137480-bcbd2f80-9842-11ea-848d-b3afb2d8ae7d.png)
Don't know why they didn't  fix it. :s
